libcodechecker/log/host_check.py

```diff
@@ -0,0 +1,38 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+
+import errno
+import subprocess
+
+from libcodechecker.logger import LoggerFactory
+
+LOG = LoggerFactory.get_new_logger('HOST CHECK')
+
+
+def check_intercept(env):
+    """
+    Simple check if intercept (scan-build-py) is available.
+    """
+    intercept_cmd = ['intercept-build', '--help']
+    try:
+        res = subprocess.call(intercept_cmd,
+                              env=env,
+                              stdout=subprocess.PIPE,
```

Comment:
We don't even need to properly communicate with `intercept-build` here, we only need to see if the command returns properly.